test/e2e/app-dir/missing-suspense-with-csr-bailout/missing-suspense-with-csr-bailout.test.ts

```diff
@@ -29,7 +29,10 @@ describe('missing-suspense-with-csr-bailout', () => {
       expect(exitCode).toBe(1)
       expect(next.cliOutput).toContain(message)
       // Can show the trace where the searchParams hook is used
-      expect(next.cliOutput).toMatch(/at.*server[\\/]app[\\/]page.js/)
+      // TODO: This path is different for Turbopack. Builds need to have sourcemaps support.
+      if (!process.env.TURBOPACK) {
+        expect(next.cliOutput).toMatch(/at.*server[\\/]app[\\/]page.js/)
```

Comment:
What's the output for turbopack?

Reply:
The error is not sourcemapped in webpack and Turbopack in this case. For Turbopack it has a file that has a hash in it (guessing hashed chunk name). In webpack it's `.next/server/app/page.js`, that's why it's relevant to skip for now. Will be better when the team gets to sourcemapping these errors.